src/test/javascript/spec/component/overview/course-conversations/layout/conversation-messages/conversation-messages.component.spec.ts

```diff
@@ -131,6 +131,25 @@ examples.forEach((activeConversation) => {
             expect(getFilteredPostSpy).toHaveBeenCalledOnce();
         }));
 
+        it('should save the scroll position in sessionStorage', fakeAsync(() => {
+            // Überwache sessionStorage.setItem
+            const setItemSpy = jest.spyOn(sessionStorage, 'setItem');
+            component.ngOnInit();
+            component.saveScrollPosition(15);
+            tick(100);
+            const expectedKey = `${component.sessionStorageKey}${component._activeConversation?.id}`;
+            const expectedValue = '15';
+            expect(setItemSpy).toHaveBeenCalledWith(expectedKey, expectedValue);
```

Comment:
could we be more specific here. This can be very helpful in debugging if there is a bug somewhere.
```suggestion
            expect(setItemSpy).toHaveBeenCalledExactlyOnceWith(expectedKey, expectedValue);
```